packaging_automation/tests/test_citus_package.py

```diff
@@ -97,7 +97,7 @@ def test_build_packages():
         assert os.path.exists(postgres_version_file_path)
         config = dotenv_values(postgres_version_file_path)
         assert config["release_versions"] == "12,13,14"
-        assert config["nightly_versions"] == "12,13,14"
+        assert config["nightly_versions"] == "13,14"
```

Comment:
Since nightly_versions for 11.x has changed, I needed to update nightly versions to fix the tests

Reply:
then should we drop 12 from L99 too ?

Reply:
Do we run the tests and assertions here? If so, I think we should see assertion failures on L99.

Reply:
No tests are being run for 10.0.3 so release version is 12,13,14 but since nightlies are running on master and master is 11.x it fails.